src/js/components/leftPanel/DocumentsTabView.tsx

```diff
@@ -1,19 +1,113 @@
-import * as React from 'react';
+import React, { useEffect, useState, Fragment } from 'react';
 import { useSelector } from 'react-redux';
+
+import { mapController } from 'js/controllers/mapController';
+
 import { RootState } from 'js/store';
 
+import { AttachmentWithURLProps } from 'js/interfaces/Attachment';
+
+import { ReactComponent as DocIcon } from 'src/images/documentIcon.svg';
+
 interface Props {
   key: string;
   label: string;
 }
 
-const DocumentsTabView = (props: Props) => {
+const DocumentsTabView = (props: Props): JSX.Element => {
+  const [allAttachments, setAllAttachments] = useState([]);
   const { activeTab, tabViewVisible } = useSelector(
     (store: RootState) => store.appState.leftPanel
   );
-
+  const { activeFeatures, activeFeatureIndex } = useSelector(
+    (store: RootState) => store.mapviewState
+  );
   const tabViewIsVisible = tabViewVisible && activeTab === props.label;
-  return <>{tabViewIsVisible && <div>Documents Tab View</div>}</>;
+
+  const [featureCollectionIndex, featureIndex] = activeFeatureIndex;
```

Comment:
```suggestion
  const [featureCollectionIndex] = activeFeatureIndex;
```
variable seems unused?